### DISCUSSION TITLE
Loading an archived PostgreSQL dump

### DISCUSSION THREAD

**Discussion by hilburn** [dbeaver/dbeaver#35994] [Q&A]
I might just being utterly dense here - I have a zipped .sql file from a PostgreSQL db and want to load it up and access the data in the tables. What's the best way to go about this?

**Comment by LonwoLonwo** [comment]
Hello @hilburn 

Maybe this article will be helpful https://github.com/dbeaver/dbeaver/wiki/Backup-Restore

**Reply by hilburn** [reply]
Thanks so much! I forgot to update this but I found the actual issue. The backup script that was dumping the database was using a word custom binary format that nothing else knew how to talk to!
